README.md

```diff
@@ -102,3 +107,16 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## 🤝 Contributing
 
 Contributions are welcome! Feel free to open issues or submit pull requests to improve Ecoute.
+
+### Installation
+
+1. `make venv`
+2. Activate the venv: `ecout_venv`
+3. `make install`
+
+### Code quality
+
+Before submitting a pull request run `make precommit` and resolve any issues. Additionally, here are some useful commands:
+
+- `make lint`
+- `make format`
```

Comment:
These are the streamlined installation changes, let me know if it is necessary to split the PR.

Reply:
Oh Sry! Got confused with the Installation section above.